src/mono/wasm/host/DevServer/ComponentsWebAssemblyApplicationBuilderExtensions.cs

```diff
@@ -1,139 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Where were we using this file and why we don't need it anymore? I can't see any other usage in the repo